pages/linux/needrestart.md

```diff
@@ -0,0 +1,36 @@
+# needrestart
+
+> Checks which daemons need to be restarted after library upgrades.
+> More information: <https://github.com/liske/needrestart>.
+
+- List outdated processes:
+
+`needrestart`
+
+- Interactively restart services:
+
+`sudo needrestart`
+
+- Run in [v]erbose or [q]uiet mode:
+
+`needrestart -{{v|q}}`
+
+- Check if the kernel is obsolete:
+
+`needrestart -k`
+
+- Check if the CPU microcode is obsolete:
+
+`needrestart -w`
+
+- Display output in batch mode:
+
+`needrestart -b`
```

Comment:
Here again, I think we shouldn't describe the flag alone, but in a specific use case.